src/main/java/command/AddEventCommand.java

```diff
@@ -0,0 +1,29 @@
+package command;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import task.*;
+import duke.*;
+
+public class AddEventCommand extends AddCommand {
+
+    private String desc;
+    private LocalDate at;
+
+    public AddEventCommand(String desc, String at) {
```

Comment:
LocalDate.parse can throw a DateTimeException, should either catch it here or make your function explicitly throw the exception.

```suggestion
    public AddEventCommand(String desc, String at) throws DateTimeException {
```